cinch/roles/jenkins_common/defaults/main.yml

```diff
@@ -7,3 +7,8 @@ jenkins_ssh_keyfile: id_rsa
 java_version: 8
 # Needed for certain OpenShift instances
 jenkins_user_uid: 1000090000
+# Set the default (empty) set of authorized SSH public keys for jenkins user.
+# To add authorized keys, override this value by adding a list of filenames
+# in the local path pointing to separate public keys.
```

Comment:
Can the given path be relative or is a full path required?  I find that sometimes Ansible needs a full path in certain environments, and we should always document whether a relative or full path is acceptable.